services/web-app/components/glossary/glossary.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Grid } from '@carbon/react'
+
+import GlossaryList from './glossary-list'
+import GlossaryNav from './glossary-nav'
+
+const Glossary = () => {
+  const glossary = require('@/data/glossary')
+  return (
+    <Grid>
+      <Column sm={4} md={8} lg={7}>
```

Comment:
```suggestion
      <Column sm={4} md={6} lg={7}>
```
to match the paragraph/component width at tablet.

Reply:
https://carbondesignsystem.com/guidelines/content/action-labels uses 8 columns at md, is it ok if we change that?
<img width="940" alt="image" src="https://user-images.githubusercontent.com/40550942/166803495-e457ae19-d33a-474a-a3cd-7e0ee35a4d47.png">

Reply:
Yea, I'm confident that's a mistake / oversight on the current website.